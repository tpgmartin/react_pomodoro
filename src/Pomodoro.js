import React from 'react'

export default class Pomodoro extends React.Component {

  constructor() {
    super()

    this._play = this._play.bind(this);
    this._pause = this._pause.bind(this);

    this.state = {
      play: false,
      time: 0,
      timeType: 0,
      title: '',
    }
  }

  _pause() {
  }

  _play() {
  }

  render() {
    return (
      <div className="controls">
      <button className="play btnIcon" onClick={this._play}></button>
      <button className="pause btnIcon" onClick={this._pause}></button>
      </div>
    )
  }
}
