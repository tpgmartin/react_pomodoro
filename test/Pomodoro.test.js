import React from 'react'
import renderer from 'react-test-renderer'
import Pomodoro from '../src/Pomodoro'

test('<Pomodoro />', () => {

  const component = renderer.create(<Pomodoro></Pomodoro>)
  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()

  tree.props.onClick()
  tree = component.toJSON()
  expect(tree).toMatchSnapshot()

})
