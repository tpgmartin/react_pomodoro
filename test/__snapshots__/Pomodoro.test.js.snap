// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pomodoro /> 1`] = `
<div
  className="controls"
>
  <button
    className="play btnIcon"
    onClick={[Function]}
  />
  <button
    className="pause btnIcon"
    onClick={[Function]}
  />
</div>
`;

exports[`<Pomodoro /> 2`] = `
<button
  className="play btnIcon"
  onClick={[Function]}
/>
`;
